### PR TITLE
Translation: Address book

### PR DESCRIPTION
**Pull request by BitDesert**

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -54,6 +54,55 @@
...
+    "you-can-also-import-your-address-book-by-using-the-url-below": "You can also import your address book by using the URL below or download it as a file.",
+    "you-can-only-track-the-balance-of-maximum-20-addresses": "You can only track the balance of maximum 20 addresses",
+    "you-can-still-import-your-address-book-by-using-the-url": "You can still import your address book by using the URL below or download it as a file.",
+    "you-can-use-the-address-book-to-store": {
+      "1": "You can use the address book to store a label for your own accounts or for easy access to other addresses throughout the application.",
+      "2": "It's also possible to track the balance and/or transactions of any address, known as \"watch-only\"."
+    },
+    "you-do-not-have-any-address-book-entries-saved-yet": {
+      "1": "You do not have any address book entries saved yet,",
+      "2": "click here to create one",
```
Comment (keerifox) [inline src/assets/i18n/en.json]: could be "2-link"

the rest looks good code changes wise, i believe

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -54,6 +54,55 @@
...
+    "import-address-book-entries-from-a-file": "Import address book entries from a file.",
+    "import-export": "Import / Export",
+    "note-exported-address-book-will-not-be-encrypted-by-your-wal": "Note: Exported address book will NOT be encrypted by your wallet password.",
+    "reload-balances": "Reload balances",
+    "scan-the-qr-code-on-any-device-to-import-your-address-book": "Scan the QR code on any device to import your address book!",
+    "show-total-tracked-balance": "Show Total Tracked Balance",
+    "successfully-deleted-address-book-entry": "Successfully deleted address book entry",
+    "the-balance-will-be-displayed-in-the-address-book": "The balance will be displayed in the address book.",
+    "this-name-is-already-in-use-please-use-a-unique-name": "This name is already in use! Please use a unique name",
+    "this-name-is-reserved-for-wallet-accounts-without-a-label": "This name is reserved for wallet accounts without a label",
```
Comment (keerifox) [inline src/assets/i18n/en.json:86]: could be reused here

https://github.com/Nault/Nault/blob/ac33875a2773a06c5e0cc341a84aa2520e5033d3/src/app/components/account-details/account-details.component.ts#L556

File: src/assets/i18n/en.json [hunk 3]
```diff
@@ -54,6 +54,55 @@
...
+    "import-address-book": "Import Address Book",
+    "import-address-book-entries-from-a-file": "Import address book entries from a file.",
+    "import-export": "Import / Export",
+    "note-exported-address-book-will-not-be-encrypted-by-your-wal": "Note: Exported address book will NOT be encrypted by your wallet password.",
+    "reload-balances": "Reload balances",
+    "scan-the-qr-code-on-any-device-to-import-your-address-book": "Scan the QR code on any device to import your address book!",
+    "show-total-tracked-balance": "Show Total Tracked Balance",
+    "successfully-deleted-address-book-entry": "Successfully deleted address book entry",
+    "the-balance-will-be-displayed-in-the-address-book": "The balance will be displayed in the address book.",
+    "this-name-is-already-in-use-please-use-a-unique-name": "This name is already in use! Please use a unique name",
```
Comment (keerifox) [inline src/assets/i18n/en.json:85]: could be reused here

https://github.com/Nault/Nault/blob/ac33875a2773a06c5e0cc341a84aa2520e5033d3/src/app/components/account-details/account-details.component.ts#L563

**Comment by keerifox** [comment]
(there are 3 more strings in account details like that, which could use translation from the address book)

File: src/assets/i18n/en.json (hunk 1 above)
Comment (BitDesert) [reply src/assets/i18n/en.json]: Changed, thanks!

**Pull request merged**